### DISCUSSION TITLE
Next step handler wait for specific id

### DISCUSSION THREAD

**Discussion by jyr2711** [westacks/telebot#57] [Q&A] [answered]
Hi all. 

I am using telebot to make a bot inside of Telegram (obvious...) On Python.

I am trying to register the Next step handler message but, as the group has many Users, some messages got in between of the bot's question and the answer, so It registers only the first message received even though It is not the response from the user that sent the command. 

I tried to filter by id but, if the first message is not valid, there is no more "listening" from the bot to check which message comes from the initial userid...

Any suggestion on how to make this work ?

Thank you

**Comment by punyflash** [comment, marked as answer]
This is a discussion of PHP library `westacks/telebot`. You need to search on your library sources, which, I assume, is [this one](https://github.com/eternnoir/pyTelegramBotAPI).
